.github/workflows/check-catalyst.yaml

```diff
@@ -250,13 +257,13 @@ jobs:
         key: ${{ runner.os }}-mhlo-${{ needs.constants.outputs.mhlo_version }}-default-source
         enableCrossOsArchive: True
         fail-on-cache-miss: True
-
-    - name: Get Cached LLVM Build
-      id: cache-llvm-build
+    
+    - name: Get Cached MHLO Build
+      id: cache-mhlo
       uses: actions/cache@v3
       with:
-        path: llvm-build
-        key: ${{ runner.os }}-llvm-${{ needs.constants.outputs.llvm_version }}-default-build-opt
+        path: mhlo-build
+        key: ${{ runner.os }}-mhlo-${{ needs.constants.outputs.mhlo_version }}-default-build
```

Comment:
I think changes here are redundant. Are we only changing the order of the "get cached mhlo" and "get cached llvm"?

Reply:
Isn't "Get Cached MHLO Build" duplicated now? (See lines below)

Reply:
Removed!